principles/3_Privacy.md

```diff
@@ -0,0 +1,51 @@
+* Users can choose how much data to share from symptom surveys
+
+## Consent
+* A member of the public (MoP) shall consent to data being collected or transmitted explicitly, that is it should not be implied by usage, or mixed with another consent
+* A MoP shall provide informed consent, that is they must be able to understand what they are consenting to and the purposes for which each category of data is gathered
+* Information about privacy shall be communicated in plain English, not legal, or technical language
+* Consent to collect data shall be separate from consent to disclose data, and include retention periods
+* Consent to collect each category of data (e.g. location, proximity) shall be separate
+* No communication initiated by 3rd parties shall be provided to a MoP, through the app, without their consent
+* No data shall be shared with 3rd parties, except healthcare authorities using the agreed mechanisms
+* The user of SafePaths shall be able to disable location tracking temporarily
+* Consent to disclose data to a HA shall be separate, at the time of disclosure, and contain a reference to the HA privacy policy
+
+## Transparency & Accountability
+
+* The privacy policy shall include the steps taken by the data collector to ensure the confidentiality, integrity, and quality of the data
+* The data controller, that is, who controls what data is collected and how it is stored and disclosed shall be identified clearly
+* No single person shall be able to modify the source code and release the application to the public
+* All source code, models and technical documentation shall be publicly available, including all dependencies
+* An assessment of any residual privacy risks shall be made public in the form of a risk assessment
+
+# Anonymization
+
+* Location, proximity, health, race/ethnicity, gender, and any data that allows the identification of a person, shall be considered sensitive unless it is sufficiently aggregated
+* Paths between waypoints shall not be stored or published
+* No identifiers of nearby devices shall be recorded that can be used to identify a MoP
+* Healthcare authorities should have the ability to effectively redact data
+* Un-redacted data shall not be persisted by a Healthcare authority in the Safe Places web application or it’s supporting components
+* Location or proximity data of less than N number of diagnosed people shall not able to be published, where N can be configured based on local legislation
+* Precautions shall be taken to ensure that shared aggregate data may not be re-identified downstream, by ensuring that no data items are passed from SafePaths to SafePlaces that are not required (see DM2)
+* Users should have the ability to effectively redact data themselves
```

Comment:
Fits better in the consent section?

Reply:
Also, does this mean we intend to provide a function for them to do this in the app, as opposed to "transmit a request to healthcare authorities to retract the data" as stated in the next section?

Reply:
There is a plan to provide the ability for the user to redact the data themselves, and to provide consent on the redacted data itself.  Moved that last bit to consent as you suggest!